test/issue-strategies/story-test.js

```diff
@@ -23,6 +41,15 @@ describe('Story/Sub-task Strategy Apply Tests', () => {
       storyStrat.apply(dummyJira.issues.SubTask12, dummyJira)
         .should.eventually.be.rejectedWith(Error, /SubTask12 is not open to commit against/)
     );
+    it('Subtask Task is yellow, but the Maintainance task parent is not', () =>
+      storyStrat.apply(dummyJira.issues.MaintenanceSubtask1, dummyJira)
+        .should.eventually.be.rejectedWith(Error, /Cannot commit.*MT5.*parent issue DispatcherLinkedSubTask4.*/)
+      );
+
+    it('Subtask Task is yellow, but the Bug task parent is not', () =>
```

Comment:
Like you did here, only move that closing parens on line 47 a couple spaces to the left.